docs/ruleset_functions/get_zone_eflh.md

```diff
@@ -0,0 +1,43 @@
+# get_zone_eflh
+
+**Description:** provides the equivalent full load hours of the zone.  Equivalent full load hours are defined as: any hour where the occupancy fraction is greater than 5% AND the HVAC system is in occupied mode.  For this function, we are recognizing the HVAC system as being in occupied mode if ANY of the HVAC systems serving the zone are in occupied mode.
+**Applicability Note:** hvac_system.fan_system.operation_schedule is being used as the HVAC operation schedule.  Therefore, this check will not work for radiant systems or other systems that do not include a fan
+
+**Inputs:** 
+- **RMI**
+- **zone**
+
+**Returns:**  
+- **flh**: a number equal to the total equivalent full load hours for the year
+ 
+**Function Call:**
+- **get_list_hvac_systems_associated_with_zone**
+
+## Logic:
+- create a dictionary to hold information about the occupancy of each space: `people_info_for_spaces = {}`
+- create a variable for the total number of zone occupants: `total_zone_occupants = 0`
+- loop through the spaces in the zone collecting information about occupancy: `for space in zone.spaces:`
+  - append the information about the occupancy to the list: `people_info_for_spaces[space]["NUM_OCCUPANTS] = space.number_of_occupants`
+  - `people_info_for_spaces[space]["SCHEDULE_HOURLY_VALUES"] = space.occupant_multiplier_schedule.hourly_values`
+  - add the occupants of this space to the total number of zone occupants: `total_zone_occupants += space.number_of_occupants`
```

Comment:
It seems that 5% occupancy criteria should be determined as follows:
1. Calculating design occupancy of each zone as a product of space.number_of_occupants and max(1, maximum occupant_multiplier_schedule fraction 
2. Determine aggregated occupancy schedule by multiplying space.number_of_occupants by occupant_multiplier_schedule fraction and then summing up the product across all zones to get 8760 occupancy schedule. 
3. For each hour, determine hourly fraction as a ratio of item 2 to item 1. 

Seems that it's aligned with what you are doing except for 1, where you do not account for the schedule value over 1.

Reply:
ok, will update